tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,11 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(rd2html("\\link[base]{library}"),
+               link_remote(label="library", topic="library", package="base"))
```

Comment:
I have but I'm afraid I repeated it incorrectly in my comment. It should be `\link[=topic]{label}` where label is whatever text you would like to appear to the user and `topic` is how you find the help document.
[R-exts](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Cross_002dreferences) gives this example `\link[=terms.object]{terms}` which should refer to the S3 "terms" class (in package stats). The relevant help topic is `terms.object` and the Rdocumentation url should be rdocumentation.org/packages/stats/topics/terms.object.